.travis.yml

```diff
@@ -7,12 +7,13 @@ sudo: false
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
+    - $HOME/.cache # includes bowers cache
 
 env:
   # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-1.13
```

Comment:
Are you OK with effectively "dropping" support for ember 1.13, or would you like me to put that back in?